src/clj_meowallet/core.clj

```diff
@@ -18,15 +18,29 @@
      "Authorization" (str "WalletPT " token)}))
 
 (def mb-ref-url "mb/pay")
+(def verify-callback-url "callback/verify")
 
 (defn prepare-data
-  [credentials data]
+  [credentials data path]
   (assoc data :host (host)
-              :path mb-ref-url
+              :path path
               :headers (headers credentials)
               :body data))
 
+(defn prepare-data-to-generate-mb-ref
+  [credentials data]
+  (prepare-data credentials data mb-ref-url))
+
 (defn generate-mb-ref
   [credentials data]
-  (-> (prepare-data credentials data)
+  (-> (prepare-data-to-generate-mb-ref credentials data)
+      (request-utils/http-post)))
+
+(defn prepare-data-to-verify-callback
+  [credentials data]
+  (prepare-data credentials data verify-callback-url))
+
+(defn verify-callback
+  [credentials data]
+  (-> (prepare-data-to-verify-callback credentials data)
```

Comment:
about the test.. actually I really don't know what to test here, because the other fns are all tested. And in this specific case the mea-wallet-api doesn't say how to test this shit